app/common/openmeter.go

```diff
@@ -64,11 +64,18 @@ func NewNamespacedTopicResolver(config config.Configuration) (*topicresolver.Nam
 	return topicResolver, nil
 }
 
-func NewKafkaIngestCollector(producer *kafka.Producer, topicResolver topicresolver.Resolver) (*kafkaingest.Collector, error) {
+func NewKafkaIngestCollector(
+	config config.Configuration,
```

Comment:
This should be limited either to ingest or ingest.kafka.

Reply:
Agreed. Fixed.